auth/src/ios/auth_ios.mm

```diff
@@ -608,5 +608,23 @@ void DisableTokenAutoRefresh(AuthData *auth_data) {}
 void InitializeTokenRefresher(AuthData *auth_data) {}
 void DestroyTokenRefresher(AuthData *auth_data) {}
 
+AuthError Auth::UseUserAccessGroup(const char* access_group_str) {
+  if (!auth_data_) {
+    return kAuthErrorUninitialized;
```

Comment:
No such error, this one doesn't exist, please check the AuthError enum and suggest a better option.